src/components/Categories.js

```diff
@@ -1,7 +1,16 @@
-const Categories = () => (
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories.js';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  return (
     <div className="categories-container">
-      <button>Check Status</button>
+      <button onClick={() => {
+        console.log(dispatch(checkStatus()));
```

Comment:
- [ ] Kindly remove the console.log in the code, we only do that for testing purposes, please check the Javascript requirements for this project for reference.